Makefile

```diff
@@ -1,22 +1,92 @@
+#! /usr/bin/make -f
+
+# Project variables.
+VERSION := development
+PROJECT_NAME := $(shell basename "$(PWD)")
 DATE := $(shell date '+%Y-%m-%dT%H:%M:%S')
-VERSION = development
 HEAD = $(shell git rev-parse HEAD)
 LD_FLAGS = -X github.com/tendermint/starport/starport/internal/version.Version='$(VERSION)' \
 	-X github.com/tendermint/starport/starport/internal/version.Head='$(HEAD)' \
 	-X github.com/tendermint/starport/starport/internal/version.Date='$(DATE)'
 BUILD_FLAGS = -mod=readonly -ldflags='$(LD_FLAGS)'
 
+# Go related variables.
+GOBASE := $(shell pwd)
+GOBIN := $(GOBASE)/dist
+GOCMD := $(GOBASE)/starport/cmd/starport
+GOCILINT := $(GOPATH)/bin/golangci-lint
+GORELEASER := $(GOPATH)/bin/goreleaser
+STARPORT := $(GOPATH)/bin/starport
+
+# For go 1.13-1.15 compatibility
+# https://maelvls.dev/go111module-everywhere/
+export GO111MODULE=on
+
+## install: Install de binary.
 install:
 	@echo Installing Starport...
 	@go install $(BUILD_FLAGS) ./...
 	@starport version
 
+## build-binary: Build the binary.
+build-binary: clean
```

Comment:
Is there any use case for this? We usually want this to get installed to our bin dir so, we can run starport easily.

Reply:
For we can use local binaries and keeps the stable binary into the bin dir. it's easier for me, but it the not a special case. We can remove it

Reply:
in fact, it is easier for me, too.  Sometimes I make special purpose starports.

Reply:
Yeah, if it's easier for you guys, let's keep it. I usually use the one lands on my bin dir because it makes the command easy to access for me.